packages/server/lib/browsers/memory/index.ts

```diff
@@ -130,25 +161,38 @@ export const getRendererMemoryUsage: () => Promise<number | null> = measure(asyn
 
     if (!process) return null
 
+    // if we found a renderer process, save it so we don't have to find it again
     rendererProcess = process
 
+    // return the memory usage for the renderer process
     return rendererProcess.memRss * KIBIBYTE
   }
 
   try {
+    // if we have a renderer process, get the memory usage for it
     return (await pid(rendererProcess.pid)).memory
   } catch {
+    // if we can't get the memory usage for the renderer process,
+    // assume it's gone and clear it out so we can find it again
     rendererProcess = null
 
     return getRendererMemoryUsage()
   }
 }, { name: 'getRendererMemoryUsage', save: true })
 
+/**
+ * Retrieves the available memory for the container/host.
+ * @returns the available memory for the container/host
```

Comment:
this also in bytes?

Reply:
Yep, updated: [47476a6](https://github.com/cypress-io/cypress/pull/25462/commits/47476a6574205785f10484cfc32d38f2859d3f6a)